platform/integrations/rancher/install-rancher-integration.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Install the Rancher integration in vCluster
+sidebar_label: Rancher Integration
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'
+import Flow, { Step } from '@site/src/components/Flow'
+
+# Install the Rancher integration
+
+The vCluster Rancher integration uses an operator to connect virtual clusters to Rancher by creating a corresponding Rancher cluster resource for each virtual cluster. It runs in the local cluster and handles registration automatically. Rancher users do not need cluster management permissions to use this integration. The operator assigns Rancher cluster owner access to the virtual cluster creator, project owners, and project members. This setup allows virtual clusters to appear and behave like managed clusters in Rancher.
+
+This guide explains how to install the vCluster Rancher Operator.
+
+## Prerequisites
+
+Ensure you have the following prerequisites before installing the vCluster Rancher Operator:
+
+- Access to the Rancher UI with admin or cluster-manager permissions
+- A running Rancher local cluster selected for installation
+- Permission to add Helm repositories and install charts
+- vCluster installed and at least one connected downstream cluster where vClusters will be deployed.
+
+## Install the vCluster Rancher Operator
+
+<Flow id="install-vcluster-rancher-operator">
+  <Step>
+  In the Rancher UI, select the **local cluster** from the cluster overview.
+  </Step>
+  <Step>
+  In the sidebar, go to **Apps → Repositories** and click **Create**.
+  </Step>
+  <Step>
+  Set a repository name and use the following Index URL: `https://charts.loft.sh`
+  </Step>
+  <Step>
+  (Optional) If pre-release charts are needed, go to **User Preferences**, scroll to **Charts**, and enable **Include Prerelease Versions**.
+  </Step>
+  <Step>
+  In the sidebar, go to **Apps → Charts** and search for **vCluster Rancher Operator**.
+  </Step>
+  <Step>
+  Select the chart and follow the prompts to install it in the **local cluster**.
+  </Step>
+  <Step>
+  After installation, go to **Workloads → Deployments** and confirm that the deployment `vcluster-rancher-operator` is active.
+  </Step>
+</Flow>
+
+After it is installed, the operator automatically registers any vCluster in Rancher. It creates a corresponding Rancher cluster, sets up the connection, and grants cluster owner permissions to relevant project users.
+
+
+## Use the vCluster Rancher Extension UI
+
+The vCluster Rancher Extension UI allows you to deploy virtual clusters directly from the Rancher user interface. It integrates vCluster provisioning into the Rancher workflow, so virtual clusters appear and behave like downstream clusters within Rancher.
```

Comment:
Virtual clusters already appear and behave like downstream clusters without the UI extension.

```suggestion
The vCluster Rancher Extension UI allows you to deploy virtual clusters directly from the Rancher user interface. It provides a separate UI for managing vClusters and a user experience more tailored to vClusters. The extension also labels vClusters in the Rancher Cluster Dashboard so you can distinguish them from physical clusters.
```